### PR TITLE
Temporary remove FileWriter support when exporting video

### PR DESCRIPTION
**Pull request by McGiverGim**
Workaround for https://github.com/betaflight/blackbox-log-viewer/issues/492

Since some update of our code (I suppose some new Node version or similar), the `FileWriter` `constructor.name` does not contain the `FileWriter` value anymore, now it contains `EventTarget`. This produces the behaviour of the issue.

I've opened an issue in the library to see if we can get a real fix for that: https://github.com/thenickdude/webm-writer-js/issues/31 because it is a good idea to stream to disk directly and not to memory.

But until then, this PR removes the `FileWriter` support and uses the "on memory" stream of the video.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by meoso** [comment]
i tested and it technically works, but very weird to ask for filename after processing, IMHO.

**Comment by McGiverGim** [comment]
@meoso true, I prefer to select the file first, and in this way use less memory. But the problem is outside our code and not depends of us. We can try to migrate to a different library, but I don't know if there are other options out there.

**Comment by haslinghuis** [comment]
For  reference:
https://itnext.io/using-node-js-to-read-really-really-large-files-pt-1-d2057fe76b33

**Comment by meoso** [comment]
I presume y'all don't want to go backwards, but version `0.48.4` is the last version that works as expected.
Could potentially use differing NWjs based on OS.

e.g.
```js
    var NWversion;
    switch (os.platform()) {
        case 'linux':
            NWversion ='0.48.4';
            break;
        default:
            NWversion ='0.50.2';
    }
```

**Comment by McGiverGim** [comment]
It does not only fail under Linux. Windows has the same problem.

**Review by ctzsnooze — APPROVED** [review]
Approved

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
@McGiverGim - I have lost track of whether this is needed still?

**Comment by McGiverGim** [comment]
Yes it is. I've a suggestion of code to test to see if it fixes the real problem, maybe this week I can get some time for it.

**Comment by McGiverGim** [comment]
This must be reverted. It was an alternate solution. If we merged the other this must be reverted.

**Pull request merged**
